meshtastic/interfaces/ble/interface.py

```
@@ -1384,157 +989,204 @@ def _format_bluetoothctl_address(cls, address: str) -> str:
-            `_get_existing_client_if_valid()`. This asymmetry is intentional so
-            explicit address operations are validated against the registry path.
+        """Return available management client via management collaborator."""
+        handler = self._get_management_command_handler()
+        state_lock = getattr(self, "_state_lock", None)
+        lock_is_owned = getattr(state_lock, "_is_owned", None)
+        if callable(lock_is_owned):
+            try:
+                if bool(lock_is_owned()):
+                    return handler.get_management_client_if_available_locked(address)
+            except Exception:  # noqa: BLE001 - lock probe remains best effort
+                pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)